syft/frameworks/torch/he/fv/evaluator.py

```diff
@@ -39,6 +40,24 @@ def add(self, op1, op2):
         else:
             raise TypeError(f"Addition Operation not supported between {type(op1)} and {type(op2)}")
 
+    def negate(self, ct):
+        """Negate a cipher i.e -(ct_value)
+
+        Args:
+            ct (Ciphertext): Ciphertext to be negated.
+
+        Returns:
+            A Ciphertext object with value equivalent to result of -(ct_value).
+        """
+        result = copy.deepcopy(ct.data)
+
+        for i in range(len(result)):
+            for j in range(len(result[i])):
+                for k in range(len(result[i][j])):
+                    result[i][j][k] = negate_mod(ct.data[i][j][k], self.coeff_modulus[j])
```

Comment:
Can coeff_modulus[j] get out of bounds?

Reply:
`i` is iterating on encrypted_size of ciphertext and `j` is iterating on coeff_modulus used in ciphertext, So till the ciphertext is from same encryption environment (I mean same parameters used) then it should not get out of bound.